content/docs/aws/authentication-oidc.md

```diff
@@ -0,0 +1,128 @@
++++
+title = "Authentication using OIDC"
+description = "Authentication and authorization support through OIDC for Kubeflow in AWS"
+weight = 90
++++
+
+This section shows the how to setup Kubeflow with authentication and authorization support through OIDC in Amazon Web Services (AWS). Similar to [Cognito tutorial](https://www.kubeflow.org/docs/aws/authentication/).
```

Comment:
For references to other docs on the Kubeflow website, we use relative URLs. This is necessary for the doc versioning, where we have a different website for each major version of Kubeflow. The relative URL keeps the user within the same website.

Change to:
```
[Cognito tutorial](/docs/aws/authentication/)
```

Reply:
Great point! I'll address it in the new commit.